internal/resourcemodifiers/resource_modifiers.go

```diff
@@ -1,3 +1,18 @@
+/*
+Copyright The Velero Contributors.
```

Comment:
According to https://www.apache.org/licenses/LICENSE-2.0#apply

Year is required. Thoughts?
```
Copyright [yyyy] [name of copyright owner]
```

Reply:
I just like the license can be forever.

Reply:
Rebased , add the years.

Reply:
Actually, according to the project code standards, copyright year is to be omitted.
https://velero.io/docs/v1.12/code-standards/#copyright-header

Reply:
I'm not sure what led to the code standards decision to remove the year. It probably should be there, but the fix to that is to change the code standards (a separate issue from this).

Reply:
yes, so how to handle this case , as before , i delete the year? WDYT@sseago

Reply:
Follow the code standards guide, delete the year..🥲🥲